### PR TITLE
fsstat NFSV3 operation

### PR DESCRIPTION
**Pull request by jeremyds**

### CODE REVIEW

File: lib/nfsc.js [hunk 1]
```diff
@@ -840,6 +840,28 @@ class V3 {
             return callback(null, path_or_obj_attrs, obj_attrs);
         });
     }
+
+    /**
+     * Procedure FSSTAT retrieves volatile file system state information.
+     *
+     * @param {root_fh}  file handle of the root inode of the export, returned
```
[the remark this answers is not in the document]
Comment (jeremyds) [reply lib/nfsc.js]: done

File: lib/nfsc.js [hunk 2]
```diff
@@ -840,6 +840,28 @@ class V3 {
...
+     *                   by the mount procedure.
+     * @param {function} callback(err: null || {obj_attrs: Object},
```
[the remark this answers is not in the document]
Comment (jeremyds) [reply lib/nfsc.js]: done

File: lib/nfsc.js [hunk 3]
```diff
@@ -840,6 +840,29 @@ class V3 {
...
+     * @param {root_fh}  root_fh file handle of the root inode of the export,
+     *                                returned by the mount procedure.
+     * @param {function} callback(err: null || {status: string,
+     *                                          obj_attrs: Object},
+     *                            obj_attrs: Object,
+     *                            tbytes, fbytes, abytes,
+     *                            tfiles, ffiles, afiles);
+     */
+    fsstat(root_fh, callback) {
+        this.client.fsstat3(root_fh,
```
Comment (ploki) [inline lib/nfsc.js]: we are supposed to receive the invarsec?

**Comment by ploki** [comment]
And don't forget to put your name in the header of the files you modify ;)

File: lib/nfsc.js (hunk 3 above)
Comment (jeremyds) [reply lib/nfsc.js]: done

File: lib/nfsc.js [hunk 4]
```diff
@@ -840,6 +840,30 @@ class V3 {
             return callback(null, path_or_obj_attrs, obj_attrs);
         });
     }
+
+    /**
+     * Procedure FSSTAT retrieves volatile file system state information.
+     *
+     * @param {root_fh}  root_fh file handle of the root inode of the export,
```
Comment (ploki) [inline lib/nfsc.js]: ```{Buffer}```

Comment (jeremyds) [reply lib/nfsc.js]: done

**Pull request merged**